liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/ServerXmlDocument.java

```diff
@@ -128,15 +177,15 @@ else if (locValue.startsWith("ftp:")) {
                     return true;
                 else {
                     NodeList apps = incDoc.getElementsByTagName("application");
```

Comment:
You have not indicated your intention not to handle nested server include or you forgot to handle it.